src/vm/builtins/builtin_runner/bitwise.zig

```diff
@@ -53,14 +75,1115 @@ pub const BitwiseBuiltinRunner = struct {
         };
     }
 
+    pub fn initDefault() Self {
+        return Self.init(&@as(bitwise_instance_def.BitwiseInstanceDef, .{}), true);
+    }
+
+    /// Initializes segments for the Bitwise builtin instance using the provided MemorySegmentManager.
+    ///
+    /// This function sets the base address for the BitwiseBuiltinRunner instance by adding a segment through the MemorySegmentManager.
+    ///
+    /// # Arguments
+    ///
+    /// - `self`: A pointer to the BitwiseBuiltinRunner instance.
+    /// - `segments`: A pointer to the MemorySegmentManager managing memory segments.
+    ///
+    /// # Returns
+    ///
+    /// An error if the addition of the segment fails, otherwise sets the base address successfully.
+    pub fn initSegments(self: *Self, segments: *MemorySegmentManager) !void {
+        self.base = @intCast((try segments.addSegment()).segment_index);
+    }
+
+    /// Generates an initial stack elements enforced by this BitwiseBuiltinRunner.
+    ///
+    /// This function initializes an ArrayList of MaybeRelocatable elements representing the initial stack.
+    ///
+    /// # Arguments
+    ///
+    /// - `self`: A pointer to the BitwiseBuiltinRunner instance.
+    /// - `allocator`: The allocator to initialize the ArrayList.
+    ///
+    /// # Returns
+    ///
+    /// An ArrayList of MaybeRelocatable elements representing the initial stack.
+    /// If the instance is marked as included, a single element initialized with the base address is returned.
+    pub fn initialStack(self: *Self, allocator: Allocator) !ArrayList(MaybeRelocatable) {
+        var result = ArrayList(MaybeRelocatable).init(allocator);
+        errdefer result.deinit();
+        if (self.included) {
+            try result.append(MaybeRelocatable.fromSegment(
+                @intCast(self.base),
+                0,
+            ));
+            return result;
```

Comment:
As we always return `result` we can remove this one no? And get only the `return result;` statement outside of the if condition